bin/reth/src/main.rs

```diff
@@ -52,9 +61,13 @@ fn main() {
 
     if let Err(err) =
         Cli::<EthereumChainSpecParser, EngineArgs>::parse().run(|builder, engine_args| async move {
-            let enable_engine2 = engine_args.experimental;
-            match enable_engine2 {
-                true => {
+            if engine_args.experimental {
```

Comment:
this will always print unless `--experimental false` is provided explicitly
```suggestion
            if engine_args.experimental.is_some() {
```

Reply:
right, good catch, because of the default `true`. fixed